front/src/modules/rollingStock/helpers/utils.ts

```diff
@@ -350,3 +351,28 @@ export const translateItemsList = <T>(t: TFunction, itemsList: T[], translationK
     id: item,
     label: !isNull(item) ? t(getTranslationKey(translationKey, String(item))) : t('unspecified'),
   }));
+
+/**
+ * This function divides rolling stock properties according to:
+ * - their name
+ * - the side of the form where they should be displayed.
+ * @param refList List of the properties names we need to display in this part of the form
+ * @param effortCurves List of the rolling stock effort curves
+ * @param checkCondition Indicates whether to check for the presence of an electrical mode
+ * @returns An object with 3 lists of properties for each side of the form
+ */
+export const splitRollingStockProperties = (
+  refList: string[],
```

Comment:
Maybe we can find a more explicit name, what do you think of this ?
```suggestion
  propertyNamesToDisplay: string[],
```